tests/postgres_tests/test_indexes.py

```diff
@@ -1,8 +1,33 @@
-from django.contrib.postgres.indexes import GinIndex
+from django.contrib.postgres.indexes import BRINIndex, GinIndex
 from django.db import connection
+from django.test import skipUnlessDBFeature
 
 from . import PostgreSQLTestCase
-from .models import IntegerArrayModel
+from .models import CharFieldModel, IntegerArrayModel
+
+
+class BRINIndexTests(PostgreSQLTestCase):
+
+    @skipUnlessDBFeature('has_brin_index_support')
+    def test_repr(self):
```

Comment:
OK. This test should probably also be removed as well from the `GinIndex` tests.